src/components/Toast/Toast.jsx

```diff
@@ -33,7 +35,34 @@ const getIcon = (type, icon) => {
     />
   ) : null;
 };
-const Toast = ({ open, autoHideDuration, type, icon, hideIcon, action, children, closeable, onClose, className }) => {
+
+const Toast = ({
+  open,
+  autoHideDuration,
+  type,
+  icon,
+  hideIcon,
+  action: deprecatedAction,
+  actions,
+  children,
+  closeable,
+  onClose,
+  className
+}) => {
+  const toastLinks = useMemo(() => {
+    return actions
+      ? actions
+          .filter(action => action.type === TOAST_ACTION_TYPES.LINK)
+          .map(({ type, ...otherProps }) => <ToastLink {...otherProps} />)
+      : null;
+  }, [actions]);
+  const toastButtons = useMemo(() => {
+    return actions
+      ? actions
+          .filter(action => action.type === TOAST_ACTION_TYPES.BUTTON)
+          .map(({ type, content, ...otherProps }) => <ToastButton {...otherProps}> {content} </ToastButton>)
```

Comment:
content and not text because button support display any kind of content and not text only

Reply:
Why do we need deperacted action?

Reply:
This component is exist since 12/2020 and therefore if someone using it (because monday-ui-react-core is open source) we don't want to break his project